spectool/specdata.xml

```diff
@@ -123,11 +123,11 @@ between two successive fields at the output of the decoding process (see <a href
   <element name="DisplayUnit" cppname="VideoDisplayUnit" level="4" id="0x54B2" type="uinteger" minver="1" default="0">How DisplayWidth &amp; DisplayHeight should be interpreted (0: pixels, 1: centimeters, 2: inches, 3: Display Aspect Ratio).</element>
   <element name="AspectRatioType" cppname="VideoAspectRatio" level="4" id="0x54B3" type="uinteger" minver="1" default="0">Specify the possible modifications to the aspect ratio (0: free resizing, 1: keep aspect ratio, 2: fixed).</element>
   <element name="ColourSpace" cppname="VideoColourSpace" level="4" id="0x2EB524" type="binary" minver="1" webm="0" bytesize="4">Same value as in AVI (32 bits).</element>
-  <element name="GammaValue" cppname="VideoGamma" level="4" id="0x2FB523" type="float" webm="0" range="&gt; 0">Gamma Value.</element>
-  <element name="FrameRate" cppname="VideoFrameRate" level="4" id="0x2383E3" type="float" range="&gt; 0">Number of frames per second. <strong>Informational</strong> only.</element>
+  <element name="GammaValue" cppname="VideoGamma" level="4" id="0x2FB523" type="float" webm="0" range="&gt; 0x0p+0">Gamma Value.</element>
+  <element name="FrameRate" cppname="VideoFrameRate" level="4" id="0x2383E3" type="float" range="&gt; 0x0p+0">Number of frames per second. <strong>Informational</strong> only.</element>
   <element name="Audio" cppname="TrackAudio" level="3" id="0xE1" type="master" minver="1">Audio settings.</element>
-  <element name="SamplingFrequency" cppname="AudioSamplingFreq" level="4" id="0xB5" type="float" mandatory="1" minver="1" default="8000.0" range="&gt; 0">Sampling frequency in Hz.</element>
-  <element name="OutputSamplingFrequency" cppname="AudioOutputSamplingFreq" level="4" id="0x78B5" type="float" minver="1" default="SamplingFrequency" range="&gt; 0">Real output sampling frequency in Hz (used for SBR techniques).</element>
+  <element name="SamplingFrequency" cppname="AudioSamplingFreq" level="4" id="0xB5" type="float" mandatory="1" minver="1" default="8000.0" range="&gt; 0x0p+0">Sampling frequency in Hz.</element>
```

Comment:
updated and squashed in https://github.com/Matroska-Org/foundation-source/pull/16/commits/28a63aa9c1fc7efa2e09950bb0a5c9593d7c5b21